app/controllers/proposals_controller.rb

```diff
@@ -83,11 +83,22 @@ def update
       redirect_to conference_program_proposals_path(conference_id: @conference.short_title),
                   notice: 'Proposal was successfully updated.'
     else
-      flash.now[:error] = "Could not update proposal: #{@event.errors.full_messages.join(', ')}"
+      flash[:error] = "Could not update proposal: #{@event.errors.full_messages.join(', ')}"
       render action: 'edit'
     end
   end
 
+  def toggle_favorite
+    user = User.find(params[:favourite_user_id])
```

Comment:
I've finally started working on merging this into my own fork for an event this summer. It seems to me that this allows anyone to toggle anyone else's favorite events. Should it not just use `current_user`? Since you should be logged in and have a cookie?

I can't really see a use case as well for someone else (such as an admin) toggling another user's favorites so I think the `favourite_user_id` parameter could probably just be removed, but I might be missing something.